src/test/scala/vct/helper/SimpleProgramGenerator.scala

```diff
@@ -0,0 +1,26 @@
+package vct.helper
+
+import vct.col.ast.Constant._
+import vct.col.ast.{ApplicableContract, Class, FileOrigin, InputOrigin, InstanceMethod, Origin, Program, Statement, TVoid}
+
+import java.nio.file.Paths
+
+object SimpleProgramGenerator {
+
+  def generateSimpleInputOrigin(): InputOrigin ={
+    FileOrigin(Paths.get(""), 1, 1, 1, 1)
+  }
+
+  def generateSimpleApplicableContract()(implicit origin: Origin = generateSimpleInputOrigin()): ApplicableContract ={
+    ApplicableContract(BooleanValue(value = true),BooleanValue(value = true),BooleanValue(value = true),Seq(),Seq(),Seq())
+  }
+
+  def generateProgramWithSingleClassAndSingleMethod(body: Statement)(implicit origin: InputOrigin = generateSimpleInputOrigin()): Program ={
```

Comment:
If you make this method local, you can call it just "procedure" or "method", like Pieter did in the Viper unit tests as well